### PR TITLE
Add 10-to-11 change log.

### PR DESCRIPTION
**Pull request by chris-wood**
Since Padding is still being discussed, I figured we'd leave related issues out of this version and take this opportunity to mark the completion of most HRR issues.

cc @davidben, @martinthomson, @cjpatton, @cbartle891, @ekr

### CODE REVIEW

**Review by cjpatton — APPROVED** [review]
I would note which changes are wire breaking, but otherwise looks good.

File: draft-ietf-tls-esni.md [hunk 1]
```diff
@@ -1656,6 +1656,31 @@ important ideas and contributions.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-tls-esni-10
+
+- Make HRR confirmation and ECH acceptance explicit (#422, #423)
+
+- Relax computation of the acceptance signal (#420, #449)
+
+- Fix ClientHelloOuterAAD generation (#438, #442)
```
Comment (davidben) [inline draft-ietf-tls-esni.md:1665]: Maybe s/Fix/Simplify/? The old version wasn't broken. Just this new one's hopefully simpler.

**Review by davidben — APPROVED** [review]

File: draft-ietf-tls-esni.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-tls-esni.md:1665]: Yep, that's better!

Comment (chris-wood) [inline draft-ietf-tls-esni.md:1665]:
```suggestion
- Simplify ClientHelloOuterAAD generation (#438, #442)
```

**Pull request merged**